packages/framework/src/Resources/config/services.yml

```diff
@@ -985,3 +985,21 @@ services:
     Shopsys\FrameworkBundle\Model\Order\Mail\OrderMail: ~
 
     Shopsys\FrameworkBundle\Model\Product\Listing\RequestToOrderingModeIdConverter: ~
+
+    productAdvancedSearchQueryBuilderExtender:
```

Comment:
there is something like convention I would say for naming such services, I would try to stick to it - i.e. not to use camelCase...

in `services.yml`, see

- `shopsys.shop.component.flash_message.sender.admin` & `shopsys.shop.component.flash_message.sender.front`
- `shopsys.shop.component.flash_message.bag.front` & `shopsys.shop.component.flash_message.bag.admin`
- `shopsys.microservice_client.product_search` & `shopsys.microservice_client.product_search_export`
- ...

Reply:
you are right, I tried to do it better